### PR TITLE
Replace Coffee example by Babel one

### PR DESCRIPTION
**Pull request by denysdovhan**
Let's say you want to try the new fancy **ECMAScript** thing. Maybe your OCD would be eased by conforming to 66 **ESLint** rules. It doesn't matter, [more than 50 Brunch plugins](http://brunch.io/plugins) aim to help with all kinds of crazy cases.

Install Babel plugin by executing `npm install --save-dev babel-brunch`. Hmm...that's exactly the same command from our previous step. Whatever. Let's create the `morning.js` file with some content:

```
$ echo 'console.log(`Good morning, ${process.env.USER}!`)' > app/morning.js
```

Since you've added Babel plugin to the board, Brunch would *magically* recompile the app and include *compiled* content of `morning.js` in the output file. **Voila!**

### CODE REVIEW

File: app/assets/_docs/getting-started.md [hunk 1]
```diff
@@ -106,18 +106,17 @@ output exactly what you've entered here. jQuery is up and running now.
...
 to 66 **ESLint** rules. It doesn't matter, [more than 50 Brunch plugins](http://brunch.io/plugins) aim to help with all kinds of crazy cases.
 
-Install CoffeeScript plugin by executing `npm install --save-dev coffee-script-brunch`.
-Hmm...that's exactly the same command from our previous step. Whatever. Let's create the `drink.coffee` file with some content:
+Install Babel plugin by executing `npm install --save-dev babel-brunch`.
+Hmm...that's exactly the same command from our previous step. Whatever. Let's create the `morning.js` file with some content:
 
 ```
-$ echo 'console.log "This is madness. Madness? This is Coffee!"' > app/drink.coffee
+$ echo 'console.log(`Good morning, ${process.env.USER}!`)' > app/morning.js
```
Comment (shvaikalesh) [inline app/assets/_docs/getting-started.md]: Deppack currently does not substitute `process.env` except `NODE_ENV`, so think may confuse people.

File: app/assets/_docs/getting-started.md [hunk 2]
```diff
@@ -106,18 +106,17 @@ output exactly what you've entered here. jQuery is up and running now.
 
 ## Plugging alternative tableware
 
-Let's say you want to try the new fancy **CoffeeScript** thing. Maybe your OCD would be eased by conforming
+Let's say you want to try the new fancy **ECMAScript** thing. Maybe your OCD would be eased by conforming
```
Comment (shvaikalesh) [inline app/assets/_docs/getting-started.md]: Maybe let's say `ECMAScript 201X`?

File: app/assets/_docs/getting-started.md (hunk 1 above)
Comment (denysdovhan) [inline app/assets/_docs/getting-started.md]: So, what's better?

File: app/assets/_docs/getting-started.md (hunk 2 above)
Comment (denysdovhan) [inline app/assets/_docs/getting-started.md]: Personally, I don't like `201X` notation, but if you insist, I'm gonna change.

Comment (shvaikalesh) [inline app/assets/_docs/getting-started.md]: Let's do `ECMAScript 6` then.

File: app/assets/_docs/getting-started.md (hunk 1 above)
Comment (shvaikalesh) [inline app/assets/_docs/getting-started.md]: `[1, 2, 3].map(n => n + 1)`

File: app/assets/_docs/getting-started.md (hunk 2 above)
Comment (denysdovhan) [inline app/assets/_docs/getting-started.md]: We're using ES7/2016 feature `**` in example. Is it OK?

Comment (shvaikalesh) [inline app/assets/_docs/getting-started.md]: That's why I proposed `ECMAScript 201X`.

Comment (denysdovhan) [inline app/assets/_docs/getting-started.md]: done

File: app/assets/_docs/getting-started.md (hunk 1 above)
Comment (denysdovhan) [inline app/assets/_docs/getting-started.md]: done

**Pull request merged**